config/routes.rb

```diff
@@ -43,6 +43,7 @@
   match '/catalog/:id/report_issue' => 'catalog#report_issue', via: [:get, :post], as: 'report_issue_solr_document'
 
   get '/browse', controller: :browse, action: :index
```

Comment:
Does it make sense to make this: 

`get '/browse/call_number', controller: browse, action: :call_number` ?

It seems to make a little more sense to me to have all the different browse actions be located at `/browse/<type>`, but I also understand the reasoning of considering call number browse to be the "default" type of browse, so `index` does make sense as well.

Reply:
yep. I was saving that for later.